lib/elixir/lib/file.ex

```diff
@@ -166,6 +166,11 @@ defmodule File do
   It can be regular file, directory, socket, symbolic link, named pipe or device file.
   Returns `false` for symbolic links pointing to non-existing targets.
 
+  ## Options
+
+  The list of options currently supports a single option, the atom `:raw`,
+  which bypasses the file server and only checks for the file locally.
```

Comment:
Please use a markdown list for the options like we usually do - even though it's just a single option.